### PR TITLE
Provide better pre-requisite for BLE_Examples.

### PR DESCRIPTION
**Pull request by pan-**
Details about software which should be installed have been added.
Link to serial drivers needed have been added.

### CODE REVIEW

**Comment by pan-** [comment]
@LiyouZhou Could you review this please ?

File: README.md [hunk 1]
```diff
@@ -7,16 +7,27 @@ Please browse to subdirectories for specific documentation.
...
+* If the OS used is Windows, the serial driver of the board has to be correctly installed.
+	* For boards with mbed interface firmware the installation instructions are located (here)[https://developer.mbed.org/handbook/Windows-serial-configuration]
+	* For nrf51-based board with a J-Link interface  please install the J-Link *software and documentation pack* available (here)[https://www.segger.com/jlink-software.html]
 
- * A device with a radio on board, such as a Nordic nRF51-based board.
 
- * A supported target, such as the FRDM K64F, with a shield or external BLE peripheral, such as an ST shield.
+In order to use BLE in mbed OS you need one of the following hardware combinations:
+
+* A device with a radio on board, such as a Nordic nRF51-based board. If one of those boards is used,
```
Comment (LiyouZhou) [inline README.md]: What does `If one of those boards is used,` mean? I would write this section as follows:
```
* A Nordic nRF51-based board such as [nrf51dk](link) or [mkit](link). OR
* A supported target, such as the [FRDM K64F](link), with a shield or external BLE peripheral, such as an [ST shield](link).
```
* A Nordic nRF51-based board such as [nrf51dk](link) or [mkit](link). OR
* A supported target, such as the [FRDM K64F](link), with a shield or external BLE peripheral, such as an [ST shield](link).

**Comment by pan-** [comment]
@LiyouZhou I have updated the PR.

**Comment by pan-** [comment]
@LiyouZhou updated !

File: README.md [hunk 2]
```diff
@@ -26,6 +38,7 @@ A yotta `target` is a supported combination of hardware board and toolchain. Thi
 yotta target that describes your configuration. The existing supported configurations
 are described below.
 
+
 Targets for BLE
 ---------------
```
Comment (LiyouZhou) [inline README.md]: You need to change the target in this section

**Comment by LiyouZhou** [comment]
+1

**Pull request merged**